### DISCUSSION TITLE
SPAR

### DISCUSSION THREAD

**Discussion by KranID0X** [OpenFAST/openfast#1734] [Q&A]
Is the 5MW_OC3Spar_DLL_WTurb_WavesIrr about Spar-buoy / Spar-submersible or does it include both

**Comment by jjonkman** [comment]
Dear @KranID0X,

The 5MW_OC3Spar_DLL_WTurb_WavesIrr model from the OpenFAST r-test (https://github.com/OpenFAST/r-test/tree/main/glue-codes/openfast/5MW_OC3Spar_DLL_WTurb_WavesIrr) is a model of the NREL 5-MW baseline wind turbine atop the OC3-Hywind spar buoy, as documented in the following specifications report: https://www.nrel.gov/docs/fy10osti/47535.pdf.

Best regards,
